tests/test_browser.py

```diff
@@ -3367,6 +3371,29 @@ def test_modularize_network_error(self):
     os.remove('a.out.wasm')
     self.run_browser('a.html', '...', '/report_result?abort(both async and sync fetching of the wasm failed)')
 
+  def test_modularize_init_error(self):
+    test_cpp_path = path_from_root('tests', 'browser_test_init_throws.cpp')
```

Comment:
Can you put this in `tests/browser` ad call it `test_modularize_init_error.cpp` to match the test name?

Reply:
Done!